### ISSUE TITLE
MW: Game Won't Start 

### ISSUE THREAD

**Issue by K-z-m-k** [xan1242/XNFSMusicPlayer#37]
As soon as I add my own m3u playlist file, the game won't start.
I've tried adding it alongside the original empty m3u file.
Changed the 'XNFSMusicPlayer.ini' Playlist file path to match the name if i renamed the playlist something else.
Anything I could think of I tried.

The playlist had 1 song in mp3 format just to test that it works but as soon as I change that Playlist.m3u file, the game won't even start.

**Comment by redhatvishnu08** [comment]
Your Playlist.m3u is in different format that the issue. I also faced this issue.

Use "WinAmp" player to create "Player.m3u".(Don't Use "VLC" player, Because it use Unicode format of song's URL.)

The Owner also mentioned

> NOTE: M3U must be NON-UNICODE! If your path contains Unicode letters, until Unicode paths are implemented, please use non-Unicode paths (don't use special characters).

**Comment by kamild1996** [comment]
Yeah, I just couldn't get this mod to work, same issue. As soon as I provide my own M3U playlist consisting of a single FLAC file, the game no longer runs. I have placed my songs in a simple directory (G:\NFSMWCustomMusic) and made sure song files have nothing but normal A-Z letters, numbers and spaces. I have tried to use foobar2000, Mp3tag or Winamp to generate an m3u (not m3u8) playlist, even tried to change the file encoding with Notepad++ to ANSI, but nothing works.

**Comment by Loirftt** [comment]
> As soon as I add my own m3u playlist file, the game won't start. I've tried adding it alongside the original empty m3u file. Changed the 'XNFSMusicPlayer.ini' Playlist file path to match the name if i renamed the playlist something else. Anything I could think of I tried.

What software are you using to make the m3u? Try using Mp3tag. That seemed to work for me.

> I have placed my songs in a simple directory (G:\NFSMWCustomMusic) and made sure song files have nothing but normal A-Z letters, numbers and spaces.

Try putting the path in your C: drive. Mine works fine there, and my path isn't "simple" ("C:\Nfs Modding\NFSMW\Game additions\Music\Songs).
Could also be the amount of songs you're trying to put in. The game isn't great at handling a high number of songs, and mine boots up with only 15 loaded.

**Comment by redhatvishnu08** [comment]
@kamild1996 I suggest you to try with only MP3 files. To check whether the format is issue

I know that Creator mentioned this 

> A LOT of formats supported:
BASS' natively supported streaming formats: MP3/MP2/MP1/OGG/WAV/AIFF
BASS' natively supported tracker formats (same as XMPlay): mo3/xm/mod/s3m/it/mtm/umx
**BASSFLAC: FLAC**

Why could you try with MP3 files?
